FreeAPS/Sources/Services/WatchManager/WatchManager.swift

```diff
@@ -5,9 +5,16 @@ import WatchConnectivity
 
 protocol WatchManager {}
 
+enum Nutrient: Decodable {
+    case carb
+    case protein
+    case fat
+}
+
 final class BaseWatchManager: NSObject, WatchManager, Injectable {
     private let session: WCSession
     private var state = WatchState()
+    private var addCarbsStateModel = AddCarbs.StateModel()
```

Comment:
As a new to Swift, this was the trickiest part how to access the `.add()` method which performs FPU calculations.

Reply:
Removed